backend/tests/test_oai.py

```diff
@@ -65,6 +65,29 @@ def delete(self, identifier):
                 p.delete()
         except OaiRecord.DoesNotExist:
             pass
+        
+    def test_read_dump(self):
+        """
+        Reads a small example dump from BASE
+        """
+        records = self.base_oai.read_base_dump(os.path.join(self.testdir, 'data/example_base_dump'), 'base_dc')
+        # Fetch the records
+        records = list(records)
+        titles = [record.getField('title') for _, record, _ in records]
+        self.assertEqual(len(records), 20)
+        self.assertTrue(['Modularizing the Elimination of r=0 in Kleene Algebra'] in titles)
+        
+    def test_load_dump(self):
+        """
+        Loads up a dump in the DB.
+        """
+        self.base_oai.load_base_dump(os.path.join(self.testdir, 'data/example_base_dump'))
+        # Loading it a second time should be very quick
```

Comment:
"Very quick"? Can we have a metrics here? Typically, if second load is expected to be about one order of magnitude faster, we could have an extra test on timing here, to assert this.

```python
start = time.time()
self.base_oai.load_base_dump(os.path.join(self.testdir, 'data/example_base_dump'))
duration = time.time() - start
start = time.time()
self.base_oai.load_base_dump(os.path.join(self.testdir, 'data/example_base_dump'))
assert((time.time() - start) < duration / 10)
```

Reply:
Yes, why not, but I try to avoid timing tests as they are not very reliable… Travis is sometimes slower at certain times, so this is prone to generate spurious build failures, which are annoying. I just tried with a factor of 5 and it failed the first time (running Pytest from the command line) but succeeded the second (from Eclipse).

Reply:
I was thinking that relative timings would be more robust than absolute timings, but apparently it's not so convenient. :+1: for not having build failures due to unreliable CI, let's skip this.